### PR TITLE
[FR] Abandoned wasteland

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page "Abandoned wasteland".

**Notify me if the PR is merge without review, that I self-review before.**

### CODE REVIEW

File: wiki/Abandoned_wasteland/fr.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
+
+# Abandoned wasteland
+
+Le **Abandoned wasteland** est un [sous-forum](https://osu.ppy.sh/community/forums) privé utilisé pour cacher les forums et les sujets du forum de la vue de tous. Ce sous-forum est uniquement accessible aux modérateurs qui ont la possibilité de modifier les forums. Les instructions aux modérateurs le concernant sont : `Mettez les choses dont nous n'avons pas vraiment besoin ici.`.
```
Comment (Airwam) [inline wiki/Abandoned_wasteland/fr.md:10]:
```suggestion
Le **Abandoned wasteland** est un [sous-forum](https://osu.ppy.sh/community/forums) privé utilisé pour cacher les forums et les sujets du forum de la vue de tous. Ce sous-forum est uniquement accessible aux modérateurs qui ont les droits pour modifier les forums. Les instructions aux modérateurs le concernant sont : `Mettez les choses dont nous n'avons pas vraiment besoin ici.`.
```

File: wiki/Abandoned_wasteland/fr.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Les forums et les sujets sont dits "Wastelanded" lorsqu'ils seront déplacés vers ce forum.
```
Comment (Airwam) [inline wiki/Abandoned_wasteland/fr.md:12]:
```suggestion
Les forums et sujets sont dits "Wastelanded" lorsqu'ils sont déplacés vers ce forum.
```

**Review by Airwam — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Airwam Need a recheck from you

**Review by Airwam — APPROVED** [review]
Seems good to me

**Pull request merged**